libraries/Microsoft.Bot.StreamingExtensions/Payloads/Assemblers/PayloadAssembler.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.StreamingExtensions.Payloads
+{
+    internal abstract class PayloadAssembler
+    {
```

Comment:
This class doesn't do anything. I would expect a class called "Assembler" to actually.. do something. 

This class doesn't. 

Delete.